test/range/count.wren

```diff
@@ -0,0 +1,25 @@
+// Ordered range.
+IO.print((2..5).count) // expect: 4
+IO.print((3..3).count) // expect: 1
+IO.print((0..3).count) // expect: 4
+IO.print((-5..3).count) // expect: 9
+IO.print((-5..-2).count) // expect: 4
+
+// Backwards range.
+IO.print((5..2).count) // expect: 4
+IO.print((3..0).count) // expect: 4
+IO.print((3..-5).count) // expect: 9
+IO.print((-2..-5).count) // expect: 4
+
+// Exclusive ordered range.
+IO.print((2...5).count) // expect: 3
+IO.print((3...3).count) // expect: 0
+IO.print((0...3).count) // expect: 3
+IO.print((-5...3).count) // expect: 8
+IO.print((-5...-2).count) // expect: 3
+
+// Exclusive backwards range.
+IO.print((5...2).count) // expect: 3
+IO.print((3...0).count) // expect: 3
+IO.print((3...-5).count) // expect: 8
+IO.print((-2...-5).count) // expect: 3
```

Comment:
I know you're not asking me, but shouldn't the result just be the same as what it currently is without the `Range.count` implementation? `Sequence.count` should already be giving you the answer (in this case, it's 2 in both cases because 2 is never part of the range, whether the 1.9 is "inclusive" or not, so in both cases the range iterates just 0 ad 1).